src/renderers/common/PostProcessing.js

```diff
@@ -10,27 +11,73 @@ class PostProcessing {
 		this.renderer = renderer;
 		this.outputNode = outputNode;
 
+		this.defaultColorTransform = true;
```

Comment:
There may be other color transforms involved in a post-processing stack, and I doubt we want this flag to disable all of them — could I recommend the word "output" be part of this name to clarify? Perhaps `outputColorTransform`?

A common, related term in color management is Output Device Transform (ODT). Though, I'm hesitant to use that exactly here, as we're mapping to a canvas drawing buffer's color space, or perhaps a framebuffer, and not necessarily for a known 'device'.

/cc @WestLangley

Reply:
I think what the property wants to express is that there is a default color transformation in `PostProcessing` which the developer can disable when `RenderOutputPass` is used. So maybe `defaultOutputColorTransform`?

Reply:
> What the property wants to express is that there is a default color transformation in PostProcessing which the developer can disable when this step is done manually

I had this doubt too, but I believe that `outputColorTransform` can convey this idea too.